mapper/orm/src/main/java/org/hibernate/search/mapper/orm/outbox/impl/OutboxEventBackgroundExecutor.java

```diff
@@ -132,10 +135,6 @@ public HibernateOrmOutboxWorker(SessionFactoryImplementor sessionFactory) {
 					}
 					throw e;
 				}
-
-				transactionHelper.commit( session );
-
-				return CompletableFuture.completedFuture( null );
 			}
```

Comment:
Did you encounter an actual issue? Because, if you look closer, in case of rollback we re-throw the exception and thus we don't commit. So I'm not sure this changes anything?

What I don't like about this change is that in case we try to commit, but it fails, we will then try to rollback. I'd expect the transaction manager to take care of that, especially since a failed commit will throw a `RollbackException`.

Reply:
you're 100% right! Dropping this commit